groovy/gradle.groovy/nbproject/project.properties

```diff
@@ -0,0 +1,19 @@
+is.eager=true
```

Comment:
Did this pass through the license check? Surprising. I thought each file has to start with a license.